tests/datastore_emulator/fence_token_vending_test.py

```diff
@@ -1,73 +0,0 @@
-from bond_app.fence_token_vending import FenceTokenVendingMachine
```

Comment:
I initially wrote this test to make sure that we were clearing out the SA key that we were caching when getting a SA key for a user. We no longer cache this SA key, so I deleted the test.